docs/README.md

```diff
@@ -9,8 +9,8 @@
 
 [![Try it on gitpod](https://img.shields.io/badge/try-on%20gitpod-brightgreen.svg)](https://gitpod.io/#https://github.com/PrismarineJS/mineflayer)
 
-| 🇺🇸 [English](README.md) | 🇷🇺 [Russian](ru/README_RU.md) | 🇪🇸 [Spanish](es/README_ES.md) | 🇹🇷 [Turkish](tr/README_TR.md) |
-|-------------------------|----------------------------|----------------------------|----------------------------|
+| 🇺🇸 [English](README.md) | 🇷🇺 [Russian](ru/README_RU.md) | 🇪🇸 [Spanish](es/README_ES.md) | 🇪🇸 [French](fr/README_FR.md) | 🇹🇷 [Turkish](tr/README_TR.md) |
```

Comment:
If I recall correctly that's not the French flag🤣

Reply:
😅